src/app/tests/suites/certification/Test_TC_OPSTATE_1_1.yaml

```diff
@@ -0,0 +1,159 @@
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads the FeatureMap attribute from the DUT"
+      PICS:
+          OPSTATE.S.Afffc
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label: "TH reads the AttributeList attribute from the DUT"
+      PICS: OPSTATE.S.Afffb
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains:
+                  [
+                      0,
+                      1,
+                      3,
+                      4,
+                      5,
+                      65528,
+                      65529,
+                      65530,
+                      65531,
+                      65532,
+                      65533,
+                  ]
+
+    - label:
+          "TH reads from the DUT the optional attribute(CountdownTime) in the AttributeList 
+          from the DUT"
+      PICS: OPSTATE.S.A0002 && OPSTATE.S.Afffb
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [2]
+
+    - label:
+          "TH reads from the DUT the EventList attribute."
+      PICS: OPSTATE.S.Afffa
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+              contains: [0x00]
+              
+    - label:
+          "TH reads from the DUT the optional event(OperationCompletion) in EventList."
+      PICS: OPSTATE.S.E01 && OPSTATE.S.Afffa
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+              contains: [0x01]
+
+    - label: "Read the optional command(Start) in AcceptedCommandList"
+      PICS: OPSTATE.S.C00 && OPSTATE.S.Afff9
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                            1,
+                            2
+                        ]
+
+    - label: "Read the optional command(Stop) in AcceptedCommandList"
+      PICS: OPSTATE.S.C01 && OPSTATE.S.Afff9
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [1]
+
+
+    - label: "Read the optional command(Pause) in AcceptedCommandList"
+      PICS: OPSTATE.S.C02 && OPSTATE.S.Afff9
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+              contains: [
+                            0,
+                            3
```

Comment:
And this test is now saying "If the device supports Start (OPSTATE.S.C02), then it must support Pause and Resume".  Again, does the PICS just need updating to test for OPSTATE.S.C00?